fe/fe-core/src/main/java/com/starrocks/sql/optimizer/GroupExpression.java

```diff
@@ -175,16 +193,19 @@ public double getCost(PhysicalPropertySet require) {
      * @param inputProperties  List of children input properties required
      * @param cost             Cost
      */
-    public void setPropertyWithCost(PhysicalPropertySet outputProperties,
+    public boolean setPropertyWithCost(PhysicalPropertySet outputProperties,
```

Comment:
A set operation should not have a return value. Or change a name to better represent what the function means.

Reply:
I will rename in next PR